sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveMetastoreCatalog.scala

```diff
@@ -199,8 +201,12 @@ private[hive] class HiveMetastoreCatalog(sparkSession: SparkSession) extends Log
       fileType: String,
       isWrite: Boolean): LogicalRelation = {
     val metastoreSchema = relation.tableMeta.schema
-    val tableIdentifier =
-      QualifiedTableName(relation.tableMeta.database, relation.tableMeta.identifier.table)
+    // scalastyle:off caselocale
+    val catalog = relation.tableMeta.identifier.catalog.getOrElse(
+      CatalogManager.SESSION_CATALOG_NAME).toLowerCase
```

Comment:
ditto, we shouldn't do special things for catalog name only.

Reply:
ack